contracts/data-storage/andromeda-form/src/state.rs

```diff
@@ -0,0 +1,48 @@
+use andromeda_data_storage::form::SubmissionInfo;
+use andromeda_std::{amp::AndrAddr, common::MillisecondsExpiration};
+use cosmwasm_std::{Addr, Uint64};
+use cw_storage_plus::{Index, IndexList, IndexedMap, Item, MultiIndex};
+
+pub const SCHEMA_ADO_ADDRESS: Item<AndrAddr> = Item::new("schema_ado_address");
+pub const START_TIME: Item<Option<MillisecondsExpiration>> = Item::new("start_time");
```

Comment:
A bunch of these Items could be stored under one Item under the name Config

Reply:
Resolved it.